lib/ansible/playbook/handler_task_include.py

```diff
@@ -26,6 +26,8 @@
 
 class HandlerTaskInclude(Handler, TaskInclude):
 
+    VALID_INCLUDE_KEYWORDS = frozenset(('listen',) + tuple(TaskInclude.VALID_INCLUDE_KEYWORDS))
+
```

Comment:
Note, this would be better written as:
``` python
VALID_INCLUDE_KEYWORDS = TaskInclude.VALID_INCLUDE_KEYWORDS.union(('listen',))
```

Reply:
I'll get an update to devel, and cherry pick the enhancement on top of this branch.

Reply:
Updated